k6/run.sh

```
@@ -0,0 +1,180 @@
+#   DB_PASSWORD MySQL root 비밀번호 (.env 파일 참고)
+# ============================================================
+
+set -e
+
+BASE_URL="${BASE_URL:-http://localhost:8080}"
+SCENARIO="${1:-all}"
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+
+# DB_PASSWORD 필요 시 .env에서 로드
+if [ -z "$DB_PASSWORD" ] && [ -f "$SCRIPT_DIR/../.env" ]; then
+  export $(grep -E '^DB_PASSWORD=' "$SCRIPT_DIR/../.env" | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)